### PR TITLE
feat: add qbd di flag

### PR DESCRIPTION
**Pull request by Hrishabh17**
https://app.clickup.com/t/86cwmdjk2

### CODE REVIEW

File: apps/orgs/serializers.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def update(self, instance, validated):
         else:
             auth_tokens = AuthToken.objects.get(user__user_id=self.context['request'].user)
             cluster_domain = get_cluster_domain(auth_tokens.refresh_token)
-            org = Org.objects.create(name=org_name, fyle_org_id=org_id, cluster_domain=cluster_domain)
+            allow_qbd_direct_integration = validated.get('allow_qbd_direct_integration', False)
```
Comment (ashwin1111) [inline apps/orgs/serializers.py:52]: why is this required when default is false?

Comment (ashwin1111) [reply apps/orgs/serializers.py:52]: FE doesn't have to send it any time, we'll control everything in backend

Comment (ashwin1111) [reply apps/orgs/serializers.py:52]: enable / disable etc

Comment (Hrishabh17) [reply apps/orgs/serializers.py]: will we enable the flag from quickbooks_connector_api?

**Review by ashwin1111 — APPROVED** [review]
